index.js

```diff
@@ -29,9 +52,253 @@ for (const file of commandFiles) {
 
 // When the client is ready, run this code (only once)
 client.once('ready', () => {
-	console.log('Ready!');
+	console.log('Ready !');
+	
+	//Delete every row in CSV
+	fsCsv.writeFileSync("answers.csv", new Parser({
+		fields: ["announcementId", "tutorId", "selection"]
+	}).parse([]));
+
+	sendNewClientMessage(1, ["Monday 9AM", "Wednesday 2PM", "Thursday 6PM"], 10, "Maths", "GCSE", 2, 1);
```

Comment:
I thought there would be a call to your API endpoint to get new course requests but I can't find it. If there is not, can u put in it or make the structure of those objects follow the actual structure of the object returned by the APU (with date options ID).